freeipa/src/main/java/com/sequenceiq/freeipa/controller/validation/CreateFreeIpaRequestValidator.java

```diff
@@ -19,7 +19,7 @@
 @Component
 public class CreateFreeIpaRequestValidator implements Validator<CreateFreeIpaRequest> {
 
-    static final String FREEIPA_INTERNAL_ACTOR_CRN = new InternalCrnBuilder(Crn.Service.FREEIPA).getInternalCrnForServiceAsString();
+    static final String INTERNAL_ACTOR_CRN = new InternalCrnBuilder(Crn.Service.IAM).getInternalCrnForServiceAsString();
```

Comment:
So yes, the internal actor is currently a single, fixed CRN. In that fixed string the IAM is the service type. Until we change that I think that the InternalCrnBuilder should not take any parameters. It's most probably not something that you introduced or need to fix. It's just a thought.